src/index.js

```diff
@@ -0,0 +1,139 @@
+"use strict";
+const state = {
+  tempEl: document.getElementById("temp"),
+  tempConEl: document.getElementById("temp-container"),
+  tempIncrease: document.getElementById("temp-increase"),
+  tempDecrease: document.getElementById("temp-decrease"),
+  tempVal: parseInt(document.getElementById("temp").textContent),
+  landscapeEl: document.getElementById("landscape"),
+  mainTitle: document.getElementById("city"),
+  inputField: document.getElementById("city-choice"),
+  getTempButton: document.getElementById("get-current-temp"),
+  skyDropdown: document.getElementById("sky-dropdown"),
+  skyEl: document.getElementById("sky"),
+  resetCityButton: document.getElementById("reset-city"),
+}
```

Comment:
Pieces of code like this are usually found in an event handler that runs on the event DOMContentLoaded, basically once the page is html is loaded on the webpage you will run a bunch of initial logic such as grabbing elements and adding event handlers to them. It will look something like this:

```javascript 
document.addEventListener("DOMContentLoaded", function () {
    state.increaseTemp = document.querySelector("#increase-temp");
    state.decreaseTemp = document.querySelector("#decrease-temp");
    state.displayTemp = document.querySelector("#display-temp");
    state.resetButton = document.querySelector("#reset-button"); 
    state.searchButton = document.querySelector("#search-button");
 
    
    state.increaseTemp.addEventListener("click", increaseTemperature);
    state.decreaseTemp.addEventListener("click", decreaseTemperature);
    ...
}
```

This is also usually found at the bottom of the file too, that way you can put all your function definitions for event handlers above it! This also just adds an extra layer of protection when it comes to things loading in order.